apis/vshn/v1/dbaas_vshn_redis.go

```diff
@@ -151,3 +174,50 @@ type XVSHNRedisList struct {
 
 	Items []XVSHNRedis `json:"items"`
 }
+
+// GetMaintenanceDayOfWeek returns the currently set day of week
+func (n *VSHNRedis) GetMaintenanceDayOfWeek() string {
```

Comment:
Can you abstract maintenance methods? Since the definition is the same across services.

Reply:
Just had the same thought, when going through the code.

Reply:
That was not my priority, let's leave that for a future improvement, this PR is already huge enough.